apps/web/src/pages/ComptrollershipExecutivesPage/ComptrollershipExecutivesPage.tsx

```diff
@@ -0,0 +1,510 @@
+import { defineMessage, useIntl } from "react-intl";
+import UserCircleIcon from "@heroicons/react/24/outline/UserCircleIcon";
+import ArrowTrendingUpIcon from "@heroicons/react/24/outline/ArrowTrendingUpIcon";
+import UserGroupIcon from "@heroicons/react/24/outline/UserGroupIcon";
+import BookOpenIcon from "@heroicons/react/24/outline/BookOpenIcon";
+
+import { CardBasic, CardFlat, Heading, Link } from "@gc-digital-talent/ui";
+import {
+  commonMessages,
+  getLocale,
+  navigationMessages,
+} from "@gc-digital-talent/i18n";
+
+import Hero from "~/components/Hero";
+import useRoutes from "~/hooks/useRoutes";
+import teamImage from "~/assets/img/comptrollership-the-team.webp";
+import newsletterImage from "~/assets/img/comptrollership-newsletter.webp";
+import connectWithPeersImage from "~/assets/img/comptrollership-connect-with-peers.webp";
+import SEO from "~/components/SEO/SEO";
+
+const pageTitle = defineMessage({
+  defaultMessage: "Comptrollership executives talent management",
+  id: "nmokFS",
+  description:
+    "Title for the comptrollership executives talent management page",
+});
+
+const pageSubtitle = defineMessage({
+  defaultMessage:
+    "Assist the Office of the Comptroller General in gathering key insights on the financial management executive workforce to shape strategic programming, support succession planning, and strengthen our community.",
+  id: "M2MQu7",
+  description:
+    "Page subtitle for the comptrollership executives talent management page",
+});
+
+const theTeamUrl = {
+  en: "https://www.canada.ca/en/treasury-board-secretariat/corporate/organization/financial-management-community-development.html",
+  fr: "https://www.canada.ca/fr/secretariat-conseil-tresor/organisation/organisation/developpement-collectivite-gestion-financiere.html",
+} as const;
+
+const newsletterUrl = {
+  en: "https://forms-formulaires.alpha.canada.ca/en/id/cm6k7jkk9006px5694mnkatr4",
+  fr: "https://forms-formulaires.alpha.canada.ca/fr/id/cm6k7jkk9006px5694mnkatr4",
+} as const;
+
+const linkedInUrl = "https://www.linkedin.com/company/fmcd-dcgf/";
+
+export const Component = () => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+
+  return (
+    <>
+      <SEO
+        title={intl.formatMessage(pageTitle)}
+        description={intl.formatMessage(pageSubtitle)}
+      />
+      <Hero
+        title={intl.formatMessage(pageTitle)}
+        subtitle={intl.formatMessage(pageSubtitle)}
+      />
+      <div data-h2-margin="base(x3, 0)">
```

Comment:
I don't think we should be using commas in margin attributes. It does work (because of some hydrogen magic I think) but it is not actually valid css.

Reply:
We've been using them a bunch in many places 😅 
Removed from page in https://github.com/GCTC-NTGC/gc-digital-talent/pull/12789/commits/8c16228ac5ec5be8156a1ee5620f95f4d56236fc